cl/scrapers/management/commands/cl_scrape_ia.py

```diff
@@ -0,0 +1,96 @@
+# !/usr/bin/python
+# -*- coding: utf-8 -*-
+
+import os
+import json
+import argparse
+import requests
+from internetarchive import get_files, search_items
+from cl.lib.command_utils import VerboseCommand, logger
+from django.conf import settings
+
+def download_from_internet_archive(options):
+    """
+    Download cases from internet archive via case law and add write them to
+    disk.
+
+    Requires a reporter abbreviation to identify cases to download.
+
+    Opitionally pass in a volume number to download that volume only.  If no
+    Volume number provided the code will cycle through the entire reporter
+    collection on IA.
+
+    :param options:
+    :return:
+    """
+    reporter = options['reporter']
+    reporter_key = ".".join(['law.free.cap',
```

Comment:
Tweak this to no longer manipulate and require IA-style abbreviations. Also update help to provide an example.